pyracing/constants.py

```diff
@@ -17,6 +17,7 @@
 URL_HOME = 'https://members.iracing.com/membersite/member/Home.do'
 
 # THE BIG ONES
+URL_DRIVER_STATS = (mStats + '/GetDriverStats')
```

Comment:
I think we had this one previously, didn't we? I can't make it actually return anything besides an empty hash

Reply:
I just moved this one up, it was lower in the list before